### ISSUE TITLE
Code blocks do not render properly

### ISSUE THREAD

**Issue by jonathan-laurent** [diderot-edu/diderot-feedback#42]
Here is how I see code blocks in the post editor:

<img width="670" alt="Screen Shot 2020-09-05 at 10 17 17 AM" src="https://user-images.githubusercontent.com/6361331/92306990-2a5fa980-ef61-11ea-86ea-315fb07ff795.png">

Here is how students see them in the published post:

<img width="593" alt="Screen Shot 2020-09-05 at 10 17 57 AM" src="https://user-images.githubusercontent.com/6361331/92306996-35b2d500-ef61-11ea-8f3e-f43152b0162c.png">

**Comment by umutacar** [comment]
Would you prefer making both look the same?

**Comment by jonathan-laurent** [comment]
Although I think it would be good, I do not have a very strong opinion on this.
However, I think that the code blocks should stand out more in the rendered message and that my second screenshot is pretty hard to parse.

**Comment by umutacar** [comment]
should be fixed.
